components/search/index.css

```diff
@@ -84,54 +98,63 @@
-	--spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-75);
+	--spectrum-search-edge-to-visual: var(--spectrum-component-pill-edge-to-visual-75);
+	--spectrum-search-top-to-icon: var(--spectrum-component-top-to-workflow-icon-75);
+	--spectrum-search-top-to-text: var(--spectrum-component-top-to-text-75);
+	--spectrum-search-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
+
+	--mod-textfield-placeholder-font-size: var(--spectrum-font-size-75);
 }
 
 .spectrum-Search--sizeL {
 	--spectrum-search-block-size: var(--spectrum-component-height-200);
 	--spectrum-search-icon-size: var(--spectrum-workflow-icon-size-200);
 	--spectrum-search-text-to-icon: var(--spectrum-text-to-visual-200);
-	--spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-200);
+	--spectrum-search-edge-to-visual: var(--spectrum-component-pill-edge-to-visual-200);
+	--spectrum-search-top-to-icon: var(--spectrum-component-top-to-workflow-icon-200);
+	--spectrum-search-top-to-text: var(--spectrum-component-top-to-text-200);
+	--spectrum-search-bottom-to-text: var(--spectrum-component-bottom-to-text-200);
+
+	--mod-textfield-placeholder-font-size: var(--spectrum-font-size-200);
 }
 
 .spectrum-Search--sizeXL {
 	--spectrum-search-block-size: var(--spectrum-component-height-300);
 	--spectrum-search-icon-size: var(--spectrum-workflow-icon-size-300);
 	--spectrum-search-text-to-icon: var(--spectrum-text-to-visual-300);
-	--spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-300);
+	--spectrum-search-edge-to-visual: var(--spectrum-component-pill-edge-to-visual-300);
+	--spectrum-search-top-to-icon: var(--spectrum-component-top-to-workflow-icon-300);
+	--spectrum-search-top-to-text: var(--spectrum-component-top-to-text-300);
+	--spectrum-search-bottom-to-text: var(--spectrum-component-bottom-to-text-300);
+
+	--mod-textfield-placeholder-font-size: var(--spectrum-font-size-300);
 }
 
-@media (forced-colors: active) {
-	.spectrum-Search .spectrum-Search-textfield,
-	.spectrum-Search .spectrum-Search-textfield .spectrum-Search-input {
-		--highcontrast-search-color-default: CanvasText;
-		--highcontrast-search-color-hover: CanvasText;
-		--highcontrast-search-color-focus: CanvasText;
-		--highcontrast-search-color-disabled: GrayText;
+/* Standard / Default */
+.spectrum-Search {
+	display: inline-block;
+	position: relative;
+	inline-size: var(--mod-search-inline-size, var(--spectrum-search-inline-size));
+	min-inline-size: var(--mod-search-min-inline-size, var(--spectrum-search-min-inline-size));
+
+	.spectrum-HelpText {
+		margin-block-start: var(--mod-search-to-help-text, var(--spectrum-search-to-help-text));
 	}
 
-	.spectrum-Search .spectrum-Search-clearButton {
-		.spectrum-ClearButton-fill {
-			/* Avoid button background color overlapping on border. */
-			forced-color-adjust: none;
-			background-color: transparent;
-		}
+	& > .is-focused.is-hover .spectrum-Search-input {
```

Comment:
I noticed when looking over the metadata.json that we have a selector using a storybook only class `.is-hover`. I'm guessing that this selector needs to change to use `:hover`.

Reply:
@jawinn fantastic catch! https://github.com/adobe/spectrum-css/pull/2673/commits/ecdbc746473720e2886f38572ae25456678ce2d9